src/main/java/bio/terra/cli/serialization/userfacing/UFWorkspace.java

```diff
@@ -56,6 +61,12 @@ public void print() {
     OUT.println(
         "Cloud console: https://console.cloud.google.com/home/dashboard?project="
             + googleProjectId);
+    // Unfortunately, this UFWorkspace may have been built from a Workspace object built from a
+    // WorkspaceDescription, which does not provide the resources or their count. In that
```

Comment:
nit: I found this sentence hard to follow.

Maybe this comment could just be shortened to say that this field is in the text output, but not the json output because it won't be consistently present.

But I was surprised to see it removed from the 'describe' json.  As a developer if the text output had extra fields not present in the JSON that would be annoying if I wanted those.  I was expecting this PR to only remove the field from the "list" JSON.

It seems like having the field missing from "describe" is perhaps worse than having it be "0" in the list?

Reply:
In order to have a different output for describe vs list, I need to introduce a new backing type, like `UFWorkspaceLight` that has everything except the `numResources`. I'd have no real way of ensuring the real `UFWorkspace` is always built with the resources populated (at least under maintenance), and this adds some complexity to the structure, but it's doable. I don't know if I can safely have `UFWorkspace extends UFWorkspaceLight` because of the Jackson builder stuff.